prober/icmp.go

```diff
@@ -118,11 +128,20 @@ func ProbeICMP(ctx context.Context, target string, module config.Module, registr
 		if srcIP == nil {
 			srcIP = net.ParseIP("0.0.0.0")
 		}
-		icmpConn, err := net.ListenPacket("ip4:icmp", srcIP.String())
+		icmpConn, err := icmp.ListenPacket("ip4:icmp", srcIP.String())
```

Comment:
Why are you changing this?

Reply:
Otherwise i got 
```prober/icmp.go:115:20: icmpConn.IPv6PacketConn undefined (type net.PacketConn has no field or method IPv6PacketConn)```

Reply:
This is the v4 codepath though, are you sure about that?

Reply:
Ah sorry let me look into it more

Reply:
Sorry I got ```prober/icmp.go:138:20: icmpConn.IPv4PacketConn undefined (type net.PacketConn has no field or method IPv4PacketConn)```

Reply:
Do you see any issue that can caused due to this change ?

Reply:
No, I was just wondering why it had changed as it appeared unrelated.